spec/models/manageiq/providers/openstack/cloud_manager/refresh_spec_common.rb

```diff
@@ -685,9 +685,9 @@ def assert_specific_vm(vm_name, attributes)
         expect(vm.private_networks.map(&:name)).to match_array vm_expected[:__network_names]
         expect(vm.public_networks.first.floating_ips).to include vm.floating_ips.first
         vm.network_ports.each do |network_port|
-          if network_port.public_network.floating_ips.count > 0
-            expect(network_port.public_network.floating_ips).to include network_port.floating_ip
-            expect(network_port.public_network.floating_ips).to include network_port.floating_ips.first
+          if network_port.public_networks.first.floating_ips.count > 0
```

Comment:
Does this assume you always have at least one public network?  Could that be a problem, or are you guaranteed one in the specs.